cli/cmd/encore/eject.go

```diff
@@ -34,8 +34,20 @@ func init() {
 		},
 	}
 
+	lang, err := appfile.AppLang(p.AppRoot)
+	if err != nil {
+		lang = appfile.LangGo
+	}
+	p.BaseImg = "scratch"
+	if lang == appfile.LangTS {
+		p.BaseImg = "node"
```

Comment:
Should this be `node:latest`?

Reply:
👍  Latest is the default tag, but probably good to be explicit